contracts/HydraDelegation/modules/VestedDelegation/VestedDelegation.sol

```diff
@@ -308,35 +308,34 @@ abstract contract VestedDelegation is
 
         if (reward == 0) return;
 
-        uint256 delegatorReward;
         uint256 stakerCommission;
+        if (position.commission != 0) {
+            (stakerCommission, reward) = _applyCommission(reward, position.commission);
+        }
+
         // Handle additional rewards if the vesting period has ended
         if (block.timestamp >= position.end + position.duration) {
             uint256 baseCommission = delegationCommissionPerStaker[staker];
```

Comment:
base commission can be declared exactly before its first usage

Reply:
Is there some benefit of doing this, or it's just a good practice?